secrets_manager/variables.tf

```diff
@@ -0,0 +1,30 @@
+variable "description" {
+  default = "terraform-managed secret"
+  type    = string
+}
+
+variable "kms_key_id" {
+  default     = null
+  description = "Optional. The KMS Key ID to encrypt the secret. KMS key arn or alias can be used."
+}
+
+variable "name" {
+  description = "Name of secret to store"
+  type        = string
+}
+
+variable "secret_value" {
+  description = "Secret value to store"
+  type        = string
+  sensitive = true
+}
+
+variable "user_tags" {
+  description = "User tags"
+  type = map(string)
+}
+
+variable "octopus_tags" {
+  description = "Octopus Tags"
+  type = map(string)
+}
```

Comment:
Why aren't you specifying the tags required as the object type?
https://github.com/variant-inc/lazy-terraform/blob/6ab03cd81fc692cf7ec81d0bdf68ec32124ca3a4/submodules/tags/variables.tf#L1-L8

https://github.com/variant-inc/lazy-terraform/blob/6ab03cd81fc692cf7ec81d0bdf68ec32124ca3a4/submodules/tags/variables.tf#L15-L21